streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -286,7 +286,7 @@ public void close(final Map<TopicPartition, Long> ackedOffsets) throws Processor
                         if (firstException == null) {
                             firstException = new ProcessorStateException(String.format("%s Failed to close state store %s", logPrefix, entry.getKey()), e);
                         }
-                        log.error("{} Failed to close state store {} due to {}", logPrefix, entry.getKey(), e);
+                        log.error("{} Failed to close state store {} due to ", logPrefix, entry.getKey(), e);
```

Comment:
Should we remove ` due to` as well or replace with a `:` since the exception will be printed in a new line?